CHANGELOG.md

```diff
@@ -1,51 +1,57 @@
 # Changelog
 
+## 1.0.0
+
+### Breaking Change
+
+- This version adds extra random digits to the URL used by the Twilio Video Apps. The format of the URL has changed from `https://video-app-xxxx-dev.twil.io` to `https://video-app-xxxx-yyyy-dev.twil.io`. The iOS and Android video apps must be upgraded in order to work with this new format. The React app does not need to be upgraded.
```

Comment:
I think we should mention here why we are doing this.

Reply:
Added